research/minigo/README.md

```diff
@@ -88,27 +87,26 @@ One iteration of reinforcement learning consists of the following steps:
  The `--evaluation` argument is to invoke the evaluation between the latest model and the current best model.
 
 ## Testing Pipeline
-As the whole RL pipeline may takes hours to train even for a 9x9 board size, we provide a dummy model with a `--debug` mode for testing purpose.
+As the whole RL pipeline may takes hours to train even for a 9x9 board size, we provide a dummy model with a `--dummy` mode for testing purpose.
```

Comment:
I think "dummy" provide less context or carry less meaningful information for user. Its hard for user to guess the meaning of the flag without look into the doc. If the flag is for testing/validation purpose, then "--test" or "--validate" is more meaningful.

Reply:
change it to --test. Thanks!